model_library/whisperx-truss/config.yaml

```diff
@@ -21,7 +21,8 @@ resources:
   memory: 14Gi
   use_gpu: true
   accelerator: T4
-secrets: {}
+secrets:
+  hf_access_token: "YOUR-HUGGING-FACE-TOKEN"
```

Comment:
let's just make this null `hf_access_token: null` for now -- it'll give us a better error message

Reply:
Updated config to reflect these suggestions